CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Express Collaborator Guide
+
+## Website Issues
+
+Open issues for the expressjs.com website in https://github.com/expressjs/expressjs.com.
+
+## PRs and Code contributions
+
+* Tests must pass.
+* Follow the [JavaScript Standard Style](https://standardjs.com/) and `npm run lint`.
+* If you fix a bug, add a test.
+
+## Branches
+
+Use the `master` branch for bug fixes or minor work that is intended for the
+current release stream.
+
+Use the correspondingly named branch, e.g. `5.0`, for anything intended for
+a future release of Express.
+
+## Steps for contributing
+
+1. [Create an issue](https://github.com/expressjs/express/issues/new) for the
```

Comment:
Since this will be fore all the repos, we probably need to fix up this section to remove the `express` specific references. Either that or we make this section "contributing to express" and add a section for the other repos below.

Reply:
I am pretty sure the best best is to remove these links. I added a not above, but I think folks may accidentally not realize it and open issues from these links in `express` when they meant to in other repos. I am just going to remove it for now and we can add it back if we figure out a good way.

Reply:
```suggestion
1. Create an issue for the
```